6 hour recurrent window change added (#112164)
Azure/azure-content: articles/virtual-machines/maintenance-configurations.md

    :::image type="content" source="./media/maintenance-configurations/add-schedule-maintenance-window.png" alt-text="Screenshot of the upper maintenance window minimum time specification.":::
- The upper maintenance window is 3.55 hours.
- A minimum of 1 hour and 30 minutes is required for the maintenance window.

> - The minimum value of recurrent window is 6 hours i.e. the value of "Repeat" should be atleast 6 hours.

Reviewer: ```suggestion
- The value of **Repeat** should be at least 6 hours.
```
